index.html

```diff
@@ -244,14 +244,91 @@ <h2>
               </div>
             </li>
           </ul>
-          <ul class="info">
-            <a href="#">Frameworks</a>
-          </ul>
-          <ul class="info">
-            <a href="#">Skills</a>
-          </ul>
+          <section class="grids">
+            <div class="grid">
+              <img class="grid-image" src="./images/frameworks.png" alt="" />
+              <h3>Frameworks</h3>
+              <div class="txt1">
+                <div class="tags1">
+                  <li class="tag1">Bootstrap</li>
+                  <li class="tag1">Ruby on Rails</li>
+                </div>
+              </div>
+            </div>
+            <div class="grid">
+              <img class="grid-image" src="./images/skills.png" alt="" />
+              <h3>Skills</h3>
+              <div class="txt1">
+                <div class="tags1">
+                  <li class="tag1">Codekit</li>
+                  <li class="tag1">Github</li>
+                  <li class="tag1">Codepen</li>
+                  <li class="tag1">Gitlab</li>
+                </div>
+              </div>
+            </div>
+          </section>
         </section>
       </section>
     </section>
+    
+    </div>
+    <section class="bg_color">
+      
+      <div class="contact_section">
+        <div class="contact_form">
+          <header>
+            <h3>Contact me</h3>
+          </header>
+          <div class="text3">
+            
+            <h2>
+              If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it.
+            </h2>
+          </div>
+          <form     
+          action="https://formspree.io/f/xpznodvr"
+          method="post"
+        >
+          <ul>
+            
+            <li>
+              <input
+              class="text"
+                type="text"
+                name="fname"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
+            </li>
+            <li>
+            <label>
+              <input
+              class="emailtext"
+                type="email"
+                name="email"
+                placeholder="Email address"
+                required
+              />
+            </label></li>
+            <li>
+              <textarea
+              class="textarea"
+                name="message"
+                placeholder="Enter your text here"
+                maxlength="500"
+                required
+              ></textarea>
+            </li>
+            <button type="submit" name="button">Get in touch</button>
```

Comment:
- [ ] Kindly add different styles for different states of the button as per the Figma template.

**Figma template**
![Screenshot from 2022-12-09 22-23-56](https://user-images.githubusercontent.com/17243325/206783532-e2f2a3da-7fcc-4ca1-aed0-207a233c284e.png)


- [ ] [OPTIONAL] You can remove the `name` attribute from this button because we only need the contents of the inputs to be collected by the Formspree service. 😎️